source/AI.cpp

```diff
@@ -1433,7 +1433,7 @@ void AI::MoveIndependent(Ship &ship, Command &command) const
 		int jumps = ship.JumpsRemaining(false);
 		// Each destination system has an average priority of 10.
 		// If you only have one jump left, landing should be high priority.
-		int planetWeight = jumps ? (1 + 40 / jumps) : 1;
+		int planetWeight = jumps ? 41 : 1;
```

Comment:
This is no longer doing what the comment suggests. Before, the less jumps a ship had remaining, the more likely it was to land. Now it just always has a high chance of landing.

Reply:
No, it isn't necessary, but it didn't return an accurate number of jumps before. It calculated the number of jumps left from only the fuel required for next jump.